assets/scss/style.scss

```diff
@@ -3428,3 +3428,30 @@ li.tab .badge {
   text-align: center;
   padding: 1rem;
 }
+
+.registration-code-list {
```

Comment:
Could you split this into its own file called like `assets/scss/settings/registration.scss` so we can eventually split this massive unsorted file into something more modular? Then include it at the top of this one with [`@forward`](https://sass-lang.com/documentation/at-rules/forward/).